domain/entity/__init__.py

```diff
@@ -11,4 +11,4 @@
 from .question import Question
 from .password_retrieval import PasswordRetrieval
 from .unavailability_time import UnavailabilityTime
-from .chatbot_input import ChatbotInput
\ No newline at end of file
+from .chatbot_input import ChatbotInput
```

Comment:
There shouldn't be a change here. Maybe you've put in a character from a keyboard in another language?

See if you can revert changes to this file.